src/Utils/FocusedMapUtils/assembleFocusedLayout.js

```diff
@@ -0,0 +1,22 @@
+import tileExampleData from './tileExampleData';
+import { insertFocusedMapPadding } from './insertFocusedMapPadding';
+
+const GRID_ROWS = 15;
+const GRID_COLS = 26;
+
+const paddedFocusedMap = insertFocusedMapPadding(tileExampleData);
```

Comment:
please move into the function.
also change grid rows to tileExampleData length and cols to tileExampleData's mid row's length